tutorials/notebooks/imx500_notebooks/pytorch/pytorch_yolov8n_seg_for_imx500.ipynb

```diff
@@ -428,6 +428,8 @@
    "source": [
     "from tutorials.mct_model_garden.models_pytorch.yolov8.yolov8 import seg_model_predict\n",
     "from tutorials.mct_model_garden.evaluation_metrics.coco_evaluation import evaluate_yolov8_segmentation\n",
+    "device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')\n",
```

Comment:
Can you use the following code, to align with other tutorials (see pytorch_yolov8n_for_imx500.ipynb):
# Get working device
device = get_working_device()